Gemfile

```diff
@@ -2,37 +2,45 @@ source "https://rubygems.org"
 
 ruby "2.3.1"
 
-gem "acts-as-taggable-on"
+gem "active_record_union"
+gem "acts-as-taggable-on", github: "mbleigh/acts-as-taggable-on"
+gem "autoprefixer-rails"
+gem "business_time"
 gem "bourbon"
 gem "clockwork"
 gem "daemons" # for delayed_job
 gem "delayed_job_active_record"
+gem "flutie"
 gem "foreman"
+gem "holidays"
 gem "jquery-rails"
 gem "kaminari"
 gem "neat"
+gem "normalize-rails", "~> 3.0.0"
 gem "omniauth-myusa"
-gem "paranoia"
+gem "paranoia", github: "rubysherpas/paranoia", branch: "core"
 gem "pg"
 gem "puma"
 gem "rack-canonical-host"
-gem "rails", "~> 4.2.0"
-gem "sass-rails"
+gem "rails", "~> 5.0.0"
```

Comment:
I like to see Rails first, and then the rest of the gems in alpha order. Basically, continuing the common practice established in the default Gemfile.